managed/models/database.go

```diff
@@ -820,9 +820,16 @@ var databaseSchema = [][]string{
 // aleksi: Go's zero values and non-zero default values in database do play nicely together in INSERTs and UPDATEs.
 
 // OpenDB returns configured connection pool for PostgreSQL.
-func OpenDB(address, name, username, password string) (*sql.DB, error) {
+func OpenDB(address, name, username, password, sslKeyPath, sslCertPath string) (*sql.DB, error) {
```

Comment:
Since we are going to support connecting to external Postgres, is there a way for us to check if the minimum Postgres version requirement is met and show a friendly error message if it's not?

Reply:
let's make current version(PG 14) as minimum

Reply:
PG version 14 check added